experiments/tootsie/exp600_tootsie.py

```diff
@@ -111,20 +120,22 @@
         train_config=llama_8b_train_config_phase2,
         tags=["llama", "8b", "ema", "exp600"],
     ),
-    override_output_path="checkpoints/lama-8b-tootsie-phase2",
+    override_output_path="checkpoints/llama-8b-tootsie-phase2",
```

Comment:
Is this a problem?

Reply:
seems to have been a stray change from before

```
(venv) (base) data_browser (fix_npm_issues)$ gsutil ls gs://marin-us-central2/checkpoints/llama-8b-tootsie-phase2
gs://marin-us-central2/checkpoints/llama-8b-tootsie-phase2/.executor_info
gs://marin-us-central2/checkpoints/llama-8b-tootsie-phase2/.executor_status
gs://marin-us-central2/checkpoints/llama-8b-tootsie-phase2/checkpoints/
gs://marin-us-central2/checkpoints/llama-8b-tootsie-phase2/hf/
(venv) (base) data_browser (fix_npm_issues)$ gsutil ls gs://marin-us-central2/checkpoints/lama-8b-tootsie-phase2
CommandException: One or more URLs matched no objects.
(venv) (base) data_browser (fix_npm_issues)$
```